File: data-structures-and-algorithms-ashrafi/python/src/recursive/problem_1/problem_1.py
def devision(a, b):
    if a < b:
        return 0
    return 1 + devision(a - b, b)


def main():
    a = 13
    b = 3

    print(f"13 / 3 = {devision(a, b)}")
    
    
if __name__ == "__main__":
    main()

File: data-structures-and-algorithms-ashrafi/java/src/recursive/problem1/Problem1.java
public class Problem1 {
    
    public static int devision(int a, int b){
        if (a < b) {
            return 0;
        }
        return 1 + devision(a - b, b);
    }

    public static void main(String[] args) {
        int a = 13;
        int b = 3;

        System.out.println(a + " / " + b + " = " + devision(a, b));
    }
}
